tests/regr_test.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+"""Run mypy on the test cases for the stdlib and third-party stubs."""
+
+from __future__ import annotations
+
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+from itertools import filterfalse, product
+from pathlib import Path
+from typing_extensions import TypeAlias
+
+from utils import (
+    PackageInfo,
+    colored,
+    get_all_testcase_directories,
+    print_error,
+    print_success_msg,
+    read_dependencies,
+    testcase_dir_from_package_name,
+)
+
+ReturnCode: TypeAlias = int
+
+SUPPORTED_PLATFORMS = ["linux", "darwin", "win32"]
+SUPPORTED_VERSIONS = ["3.11", "3.10", "3.9", "3.8", "3.7"]
```

Comment:
Maybe we should move this into `utils.py` as well? Or maybe add a `config.py`?

Reply:
I think it might be better to keep this per-test. We already have one test that supports a narrower version range than this:

https://github.com/python/typeshed/blob/e4d0d3d9d7b7366096d95ad10dada1638ceebf82/tests/typecheck_typeshed.py#L16

And we started testing 3.11 with `mypy_test.py` much earlier than we started testing it with `stubtest_stdlib.py`.